File: wertyu/wertyu.java
import java.util.HashMap;
import java.util.Map;
import java.util.Scanner;

public class wertyu {
    public static void main(String[] args) {
        Map<Character, Character> mp = new HashMap<>();
        mp.put('w', 'q');
        mp.put('e', 'w');
        // Add other character mappings here...

        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter a string: ");
        String input = scanner.nextLine();

        StringBuilder result = new StringBuilder();
        for (char c : input.toCharArray()) {
            result.append(mp.getOrDefault(c, c));
        }

        System.out.println("Mapped string: " + result);
    }
}

File: wertyu/wertyu.py
def main():
    mp = {
        'w': 'q', 'e': 'w', 'r': 'e', 't': 'r', 'y': 't', 'u': 'y', 'i': 'u', 'o': 'i', 'p': 'o',
        '[': 'p', 's': 'a', 'd': 's', 'f': 'd', 'g': 'f', 'h': 'g', 'j': 'h', 'k': 'j', 'l': 'k',
        ';': 'l', 'x': 'z', 'c': 'x', 'v': 'c', 'b': 'v', 'n': 'b', 'm': 'n', ',': 'm', ' ': ' '
    }

    inp = input("Enter string: ")
    ans = ''.join(mp.get(char, char) for char in inp)
    print("Ans:", ans)

if __name__ == "__main__":
    main()
